README.md

```diff
@@ -88,139 +77,61 @@ RUST_LOG=debug RUST_BACKTRACE=1 ./target/release/mintlayer-core -lruntime=debug
 
-The runtime in this project is constructed using many FRAME pallets that ship with the
-[core Substrate repository](https://github.com/paritytech/substrate/tree/master/frame) and a
-template pallet that is [defined in the `pallets`](./pallets/template/src/lib.rs) directory.
 
-A FRAME pallet is compromised of a number of blockchain primitives:
+### Pallets
 
--   Storage: FRAME defines a rich set of powerful
-    [storage abstractions](https://substrate.dev/docs/en/knowledgebase/runtime/storage) that makes
-    it easy to use Substrate's efficient key-value database to manage the evolving state of a
-    blockchain.
--   Dispatchables: FRAME pallets define special types of functions that can be invoked (dispatched)
-    from outside of the runtime in order to update its state.
--   Events: Substrate uses [events](https://substrate.dev/docs/en/knowledgebase/runtime/events) to
-    notify users of important changes in the runtime.
--   Errors: When a dispatchable fails, it returns an error.
--   Config: The `Config` configuration interface is used to define the types and parameters upon
-    which a FRAME pallet depends.
+Mintlayer relies on a host of Substrate pallets and a few Mintlayer specific pallets.
 
-### Run in Docker
+-   pp: The implementation of programmable pools on Mintlayer. Essentially wasm smart contracts
+-   utxo: Mintlayer's UTXO system
+    
+### Libs
 
-First, install [Docker](https://docs.docker.com/get-docker/) and
-[Docker Compose](https://docs.docker.com/compose/install/).
+Libs is home to code that is code that Mintlayer relies on but isn't technically a pallet. 
 
-Then run the following command to start a single node development chain.
+-   chainscript: Mintlayer's bitcoin script implementation.
+-   bech32: code for handling transactions with destinations encoded using bech32
 
-```bash
-./scripts/docker_run.sh
-```
+### Testing
 
-This command will firstly compile your code, and then start a local development network. You can
-also replace the default command (`cargo build --release && ./target/release/node-template --dev --ws-external`)
-by appending your own. A few useful ones are as follow.
+You'll find unit tests littered throughout the codebase but the test directory is home to the functional test framework which is heavily based on Bitcoin's functional test framework. 
 
-```bash
-# Run Substrate node without re-compiling
-./scripts/docker_run.sh ./target/release/node-template --dev --ws-external
+### Crypto
+As it stands Mintlayer uses Schnorr for all crypto-related things. There is a plan to move to our BLS implementation in the near future but this, as it stands, is a work in progress.
 
-# Purge the local dev chain
-./scripts/docker_run.sh ./target/release/node-template purge-chain --dev
-
-# Check whether the code is compilable
-./scripts/docker_run.sh cargo check
-```
+### Contributing
+[See this guide](https://github.com/mintlayer/core/CONTRIBUTING.md)
```

Comment:
I think this should be in the format https://github.com/mintlayer/core/blob/master/CONTRIBUTING.md

Reply:
The entire link spelt out? Why?

Reply:
I think the link is broken, try for yourself

Reply:
So it is.